mlflow/server/auth/__main__.py

```diff
@@ -0,0 +1,21 @@
+import click
+import sqlalchemy
+
+from .db import utils
+
+
+@click.group()
+def commands():
+    pass
+
+
+@commands.command()
+@click.option("--url", required=True)
+@click.option("--revision", default="head")
+def migrate(url: str, revision: str) -> None:
```

Comment:
the tracking migration cli is `mlflow db upgrade`, do we want to use `upgrade` command to align?

Reply:
+1